README.md

```diff
@@ -0,0 +1,12 @@
+# DLRS
```

Comment:
Would you know which readme.md wins? We have two readmes if we merge this. `readme.markdown` already exists.

Reply:
Oops. I will fix this.